app/src/test/java/com/spartahack/spartahack17/Presenter/SchedulePresenterTest.java

```diff
@@ -0,0 +1,103 @@
+package com.spartahack.spartahack17.Presenter;
+
+import android.util.Log;
+
+import com.spartahack.spartahack17.Model.Event;
+import com.spartahack.spartahack17.Retrofit.GSONMock;
+import com.spartahack.spartahack17.View.ScheduleView;
+
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeComparator;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import java.util.ArrayList;
+import java.util.Collections;
+
+import static junit.framework.Assert.assertEquals;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+/**
+ * Created by memuyskens on 10/16/16.
+ * SpartaHack-Android
+ * Unit tests for {@link SchedulePresenter}
+ */
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({Log.class, Collections.class})
```

Comment:
you should not need the log class here, if so pull it out of the presenter, also dont know if the collelctions is needed